src/zenml/utils/yaml_utils.py

```diff
@@ -20,9 +20,16 @@
 
 import yaml
 
+from zenml.constants import (
+    ENV_ZENML_ALLOW_NON_ASCII_MATERIALIZERS_DUMPS,
+    handle_bool_env_var,
+)
 from zenml.io import fileio
 from zenml.utils import io_utils
 
+ZENML_ALLOW_NON_ASCII_MATERIALIZERS_DUMPS = handle_bool_env_var(
```

Comment:
If you decide to go with the second suggestion of my other comment (keeping this a materializer-only environment variable), I think it would be nice to have it called something with a `ZENML_MATERIALIZER_` prefix. This is just to have some consistency if we decide to add more environment variables for materializer configurations.